vanillaboot.js.org

eyallior opened the pull request:
<!--

Thanks for creating a pull request to request a new subdomain from JS.ORG

Before creating your pull request, please complete the following steps:

- Ensure that your pull request changes only the cnames_active.js file, adding a single new line for your subdomain request
- Tick the two checkboxes, agreeing to the sentences, below by placing an x inside the square brackets ([ ] becomes [x])
- Add a link (GitHub repository, Vercel deployment, etc.) and explanation below for your content so we can validate your request

-->

https://github.com/eyallior/vanillaboot

Bootstrap vanilla framework that lets you develop apps with reusable, nest-able components yet stay with good old Vanilla JS.


- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)
- The site content can be seen at <link>

> The site content is ...

MattIPv4 commented on cnames_active.js:
```diff
@@ -3418,6 +3418,7 @@ var cnames_active = {
   "vanilla": "haeresis.github.io/vanilla-js-dom",
   "vanilla-picker": "sphinxxxx.github.io/vanilla-picker",
   "vanilla-swr": "droyson.github.io/vanilla-swr",
+  "vanillaboot": "github.com/eyallior/vanillaboot",
```
```suggestion
  "vanillaboot": "eyallior.github.io/vanillaboot",
```
As the CI error notes, you'll want to target the GitHub pages hostname (and path), not the repository itself

eyallior replied:
I prefer the repo, if it's ok

indus replied:
The CNAME based system we provide can only be used for the Github pages and not redirect to the repo. So please make the necessary change or we can't process your request.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```vanillaboot.js.org``` to make it work. I've seen you have set up a gh-pages branch but it seems that is not set as the source of your Github page but instead the README.md from the main branch gets rendered. You can change that in your repo settings or move the CNAME to themain branch to make it work. 
FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages

I'll merge after we have "contact". Please keep this PR open.

eyallior commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `vanillaboot.js.org` to make it work. I've seen you have set up a gh-pages branch but it seems that is not set as the source of your Github page but instead the README.md from the main branch gets rendered. You can change that in your repo settings or move the CNAME to themain branch to make it work. FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages
> 
> I'll merge after we have "contact". Please keep this PR open.

Added

indus commented:
Welcome https://vanillaboot.js.org
